src/hooks/useGeographicalStateFromRoute.ts

```diff
@@ -5,6 +5,10 @@ import {CONST as COMMON_CONST} from 'expensify-common/lib/CONST';
 type CustomParamList = ParamListBase & Record<string, Record<string, string>>;
 type State = keyof typeof COMMON_CONST.STATES;
 
+/**
+ * Extracts the 'state' (default) query parameter from the route/ url and validates it against COMMON_CONST.STATES, returning its ISO code or `undefined`.
+ * Example: const stateISO = useGeographicalStateFromRoute(); // Assuming 'state' param is 'CA' or another valid state, returns the corresponding ISO code or `undefined` if invalid.
```

Comment:
The example I meant more so an example of the the URL and what it returns than how to use it in code